app/assets/javascripts/application.js

```diff
@@ -77,7 +77,7 @@ function onContentLoad(){
   $("#title_action a").addClass('btn').not('.btn-info, .btn-danger').addClass("btn-default");
 
   $("#title_action li a").removeClass("btn btn-default").addClass("la");
-  $("#title_action span").removeClass("btn btn-default").addClass("btn-group");
+  $("#title_action span").not('.glyphicon').removeClass("btn btn-default").addClass("btn-group");
```

Comment:
I have no clue why we are using js to change these classes.
I am fine with leaving this in the PR, so up to you whether to accept it here or ask for a separate PR :smile: